### PR TITLE
Reactive context-path added to Playground

### PR DESCRIPTION
**Pull request by joaodias14**
### :pencil: Description
Reactive context-path added to Playground Route configuration.

### :link: Related Issues
https://github.com/ExpediaGroup/graphql-kotlin/issues/1097

### CODE REVIEW

File: servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt [hunk 1]
```diff
@@ -32,13 +32,22 @@ import org.springframework.web.reactive.function.server.html
 @Configuration
 class PlaygroundRouteConfiguration(
     private val config: GraphQLConfigurationProperties,
-    @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource
+    @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource,
+    @Value("server.servlet.context-path") private val servletContextPath: String?,
```
Comment (dariuszkuc) [inline servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt:36]: we only support webflux in the `spring-server` so this should be removed

File: servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt [hunk 2]
```diff
@@ -32,13 +32,17 @@ import org.springframework.web.reactive.function.server.html
 @Configuration
 class PlaygroundRouteConfiguration(
     private val config: GraphQLConfigurationProperties,
-    @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource
+    @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource,
+    @Value("spring.webflux.base-path") private val contextPath: String?
```
Comment (dariuszkuc) [inline servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt:36]: does this actually work? I think it should be `@Value("\${spring.webflux.base-path}")`

**Comment by dariuszkuc** [comment]
Thanks for fixing this!

File: servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt (hunk 2 above)
Comment (joaodias14) [reply servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/PlaygroundRouteConfiguration.kt:36]: Yes, you are right. I will update it.

**Review by dariuszkuc — APPROVED** [review]

**Pull request merged**